### ISSUE TITLE
Add str method for ddfs

### ISSUE THREAD

**Issue by ahalterman** [tesseradata/datadr#19]
It would be nice to call str() on a distributed dataframe and get back the number of key-value pairs and the standard str() information for column names/classes/lengths.

**Comment by lhsego** [comment]
Agreed.  An `str()` method for `ddo` and `ddf` objects would be nice.

**Comment by tcpulsipher** [comment]
Totally unnecessary. One can see the ddo and ddf objects easily already.
Spend your time on much more important things.
Trenton

**Comment by hafen** [comment]
The print method does a lot of this.  I agree with @tcpulsipher that it's not a huge issue, but @lhsego, if you have a specific idea of what the output of str should look like, please comment - could be a quick thing and a nice introductory contributor's task.  But there are many other things to do!
